src/features/mentee-dashboard/services/deleteApplication.ts

```
@@ -0,0 +1,37 @@
+import { api } from "@/app/lib/api";
+
+import { toast } from "@/shared/lib/toast";
+
+import { APPLICATION_QUERY_KEYS } from "./_keys";
+import { useMutation, useQueryClient } from "@tanstack/react-query";
+
+export interface DeleteApplicationResponse {}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)